rero_mef/theme/__init__.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of RERO MEF.
+# Copyright (C) 2018 RERO.
```

Comment:
Copyright (C) 2020 RERO.

Reply:
After the merge, we can change all the license to the one we want https://github.com/rero/rero-mef/issues/45.